packages/modules/src/modules/epics/utils/ophan.ts

```diff
@@ -30,6 +33,30 @@ export const OPHAN_COMPONENT_EVENT_CTAS_VIEW: OphanComponentEvent = {
     action: 'CLICK',
 };
 
+export const OPHAN_COMPONENT_EVENT_APPLEPAY_VIEW: OphanComponentEvent = {
+    component: {
+        componentType: 'ACQUISITIONS_OTHER',
+        id: OPHAN_COMPONENT_ID_APPLEPAY_VIEW,
+    },
+    action: 'CLICK',
```

Comment:
The action should be "VIEW" I believe?

Reply:
Good spot, need to check these things more closely, corrected now